src/components/autocomplete/autocomplete.scss

```diff
@@ -214,7 +214,7 @@ md-autocomplete {
   list-style: none;
   padding: 0;
 
-  li {
+  .md-autocomplete-suggestion {
```

Comment:
These styles are not applied anymore to <md-not-found> item element which is converted to "li".
You can see this in the first autocomplete demo https://material.angularjs.org/latest/demo/autocomplete

Reply:
You can see these styles applied in this screenshot using that demo:
![Screen Shot 2020-01-30 at 10 21 01](https://user-images.githubusercontent.com/3506071/73462735-41df1000-434a-11ea-915e-35300797e895.png)

Reply:
Yeah, but enter some query which doesn't match any results and you'll see that not-found template is not styled anymore.

Reply:
OK, thank you for the updated reproduction steps. I've opened https://github.com/angular/material/issues/11852 to track this.

Reply:
Thanks for opening issue :)